numenc_module.pyi

```diff
@@ -1,25 +0,0 @@
-# stubs for the IDE to provide autocompletion.
```

Comment:
Don't you need this file for IDE autocompletion and mypy?

Reply:
I just renamed it to `numenc.pyi`